src/stylesheets/utility/_tag-list.scss

```diff
@@ -1,20 +1,173 @@
 .tag-list {
   list-style-type: none;
   padding: 0;
+  position: relative;
+
+  @media (screen(lg)) {
+    height: rem(800);
+    margin-top: rem(-800);
+  }
 }
 
 .tag-list__item {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
   align-items: center;
   background-color: color(white);
   border-radius: rem(4);
-  margin: 0 0 rem(30) 0;
-  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 auto rem(30) auto;
+  max-width: rem(500);
   padding: 0 rem(12) rem(24) rem(12);
+  position: relative;
 
   a {
     color: color(coral-pink);
   }
+
+  @media (screen(lg)) {
+    box-sizing: border-box;
+    border-radius: rem(100) rem(10) rem(10) rem(100);
+    padding: rem(32) rem(22) rem(32) rem(82);
+
+    .-orange,
+    .sm-sans-serif-text {
+        text-align: left;
+      }
+
+    &.-alt {
+      border-radius: rem(10) rem(100) rem(100) rem(10);
+      padding: rem(32) rem(82) rem(32) rem(22);
+
+      .-orange,
+      .sm-sans-serif-text {
+        text-align: right;
+      }
+    }
+  }
+}
```

Comment:
Good job making this work with flexbox and not `position: absolute`. 👏 

`position: absolute` is powerful, but doesn't work well with content like this with text wrapping that changes the size/length of a box responsively.